### PR TITLE
Release preparation for v1.5.0

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Update changelog and user guide.

# Checklist
- [ ] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
👍

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

File: js-miniapp-sdk/README.md [hunk 1]
```diff
@@ -203,6 +203,24 @@ miniApp.setScreenOrientation(ScreenOrientation.LOCK_LANDSCAPE) // or LOCK_PORTRA
...
+### 7. Get access token
+
+You can get an access token provided by the Host App. The Host App will be able to deny your request if your mini app ID is not approved to access the token.
+
+This will return an object containing `token` and `validUntil` keys.
+
+```javascript
+miniApp.getAccessToken()
+  .then(data => {
+      const isValid = data.validUntil.getTime() <= Date.now();
```
Comment (minh-rakuten) [inline js-miniapp-sdk/README.md:215]: I think `>=` is more make sense.

Comment (corycaywood) [reply js-miniapp-sdk/README.md:215]: Yep 😆

**Review by minh-rakuten — APPROVED** [review]
👍 👍

**Pull request merged**